README.md

```diff
@@ -1,8 +1,23 @@
-# react-vrpc
+# VRPC Client (React)
 
-React wrapper around the [vrpc](https://github.com/bheisen/vrpc) library
+## Introduction
 
-## Step 1 - Create VrpcProvider
+The react vrpc client is a complete solution for backend-frontend communication.
+Unlike other libraries it allows to communicate with an arbitrary number
+of potentially distributed backends at the same time.
+
+The react vrpc client integrates seamlessly in an non-opinionated way
+into modern react architectures. Built on top of the
+[VRPC](https://github.com/bheisen/vrpc) library it allows to call backend code
+directly, obsoleting the need to define any wrappers, schemas, resolvers or adapters.
+
+## Getting started
+
+Maybe the easiest way to get started is by following [this example](https://vrpc.io/examples/react-app).
```

Comment:
The example is not yet reachable. Is that intended? Does the example not yet exist?